### PR TITLE
alliance

### PR DESCRIPTION
**Pull request by faddat**
## Description and Motivation

Migaloo v2 really just adds the alliance module.  We've shifted all other updates away from this release to focus on the alliance integration. 

---
## Checklist:

- [ ] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-chain/blob/main/docs/CONTRIBUTING.md).
- [ ] My pull request has a sound title and description (not something vague like `Update index.md`)
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation.
- [ ] The code is formatted properly `golangci-lint run ./... --fix`.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -987,6 +1038,15 @@
 	return dupMaccPerms
 }
 
+func stringMapKeys[V any](m map[string]V) []string {
```
[the remark this answers is not in the document]
Comment (faddat) [reply app/app.go:1041]: @ThanhNhann can you double check this?  I think it is the linter mis-checking the generic used here, but I'd like to be sure.

[the remark this answers is not in the document]
Comment (ThanhNhann) [reply app/app.go:1041]: yes, I will check now

[the remark this answers is not in the document]
Comment (faddat) [reply app/app.go:1041]: thanks very much!

[the remark this answers is not in the document]
Comment (ThanhNhann) [reply app/app.go]: it's good now

**Comment by faddat** [comment]
@ThanhNhann can you see if this affects alliance itself?  thanks!

**Review by faddat — COMMENTED** [review]
This branch adds the alliance module to migaloo-chain.

It's been reviewed by team members from both TFL and Notional.

**Pull request merged**